packages/validation/docs/configuration.md

```diff
@@ -186,13 +186,32 @@ settings:
 
 ### Resource Links
 
-Validates whether the comply with a JSON schema.
```

Comment:
This is missing a changeset, you can add it by executing `npx changeset` in the repository root and following the interactive questions